src/string.cpp

```diff
@@ -228,6 +228,9 @@ void StrTrimInPlace(std::string &str)
 	str = StrTrimView(str);
 }
 
+/**
+ * Returns a new string view with any space (' ') at the begin and end end of the string cut off.
+ */
```

Comment:
I looked at the usages of `StrTrimView` and `StrTrimInPlace`. They all could benefit from trimming `\t\r\n` as well. In particular the hotkey parsing from ini file should ignore `\t`.

So maybe unify the behavior of the 3 trim functions, and `StringConsumer::WHITESPACE_OR_NEWLINE`.
Then the new function can be named `StrRTrimView` or something similar.

Reply:
Moved the trim parts of this PR to #14190.